mitodl/mitxpro
# added ol-django-authentication app to MITxPro

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #2318 #2306

#### What's this PR do?

This PR accomplishes the following tasks
- Adds ol-django-authentication app to MITxOnline.
- Removes unused djoser code

#### How should this be manually tested?

This PR can be tested using the following steps
- Create a MITxPRO online account
- Try to change password via `Forgot Password` and verify it works
- Try to change password via `Forgot Password`, Change case of email and verify it works
- Try to change password via Account page and verify `Valid notification is sent`, `Password is changed`, `User remains signed in`
